content/2_Bronze/Complete_Rec.mdx

```diff
@@ -253,6 +279,23 @@ print(ans)
 </PySection>
 </LanguageSection>
 
+Note the fancy bitwise operations:
```

Comment:
why did you move this down here?

Reply:
intended flow imo should be
"oh alr look @ the code"
"wtf are these things"
scrolls down
"oh alr"

Reply:
nah what if they dont even realize its there
its awkward to put it all the way down there too
whats the advantage of this? They might figure out what the operation is doing by themselves? They're pre-bronze contestants, after all.

Reply:
fair